superset/assets/javascripts/dashboard/components/SliceCell.jsx

```diff
@@ -42,10 +42,20 @@ function SliceCell({ expandedSlices, removeSlice, slice }) {
               >
                 <i className="fa fa-pencil" />
               </a>
-              <a href={getExploreUrl(slice.form_data, 'csv')} title="Export CSV" data-toggle="tooltip">
+              <a
+                className="exportCSV"
+                href={getExploreUrl(slice.form_data, 'csv')}
+                title="Export CSV"
+                data-toggle="tooltip"
+              >
```

Comment:
Are these lines the same as in master, but formatted differently?

Reply:
`className` was added, and eslint wants the code formatted this way when having it on one line would exceed 100 characters